flink-runtime/src/main/java/org/apache/flink/runtime/externalresource/ExternalResourceUtils.java

```diff
@@ -46,12 +49,19 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ExternalResourceUtils.class);
 
+    // Indicate an empty external resources list
+    public static final String EMPTY = "empty-list";
```

Comment:
I'd suggest to:
- Use the phrase `none`
- Add the constant in `ExternalResourceOptions`
- Mention in the description of `EXTERNAL_RESOURCE_LIST` that the keyword is preserved